app/components/Form/ProjectManagerForm.tsx

```diff
@@ -51,8 +51,8 @@ const ProjectManagerForm: React.FC<Props> = (props) => {
           edges {
             node {
               rowId
-              firstName
-              lastName
+              givenName
+              familyName
```

Comment:
Everywhere where both `givenName` and `familyName` are used, you want to use `fullName` instead